### ISSUE TITLE
inconsistency in npm

### ISSUE THREAD

**Issue by zizai** [hardchor/electron-redux#15]
There are console messages in the npm build -- inconsistent with the current release.

**Comment by hardchor** [comment]
Hi. What messages are you seeing?

**Comment by zizai** [comment]
For example, after installing your timesheets app, I see this in `app/node_modules/electron-redux/dist/forwardToMain.js`
```javascript
var forwardToMain = function forwardToMain(store) {
  return function (next) {
    return function (action) {
      // eslint-disable-line no-unused-vars
      console.log('##### forwardToMain', action);
      // return next(action);
      if (!(0, _validateAction2.default)(action)) return next(action);

      if (action.type.substr(0, 2) !== '@@' && action.type.substr(0, 10) !== 'redux-form' && (!action.meta || !action.meta.scope || action.meta.scope !== 'local')) {
        _electron.ipcRenderer.send('redux-action', action);

        // stop action in-flight
        // eslint-disable-next-line consistent-return
        return;
      }

      // eslint-disable-next-line consistent-return
      return next(action);
    };
  };
};
```
But that console line is not in the source.

**Comment by hardchor** [comment]
@zizai can you double-check what version of `electron-redux` you have installed (and what version of `electron` as well, please)?

**Comment by zizai** [comment]
@hardchor Testing with 1.2.4... When I build it on my own, it's fine. But downloading from npm, I still see console logs.

**Comment by hardchor** [comment]
@zizai Apologies, that was my mistake! I published without compiling the project first. I've added some tooling around that to avoid that in the future and have published 1.2.5 with the fixes.

**Comment by zizai** [comment]
@hardchor Great. Thanks!
